docker/topic_init.sh

```diff
@@ -1,23 +1,28 @@
 #!/bin/bash
 
+MAX_RETRIES=5
```

Comment:
Please make this variable accept environment variable, e.g.
```suggestion
MAX_RETRIES=${TOPIC_INIT_RETRIES:-5}
```